pyhap/accessory_driver.py

```diff
@@ -502,6 +518,9 @@ def stop(self):
         """
         # TODO: This should happen in a different order - mDNS, server, accessory. Need
         # to ensure that sending with a closed server will not crash the app.
+        if self.run_sentinel.is_set():
```

Comment:
could you elaborate on this?

Reply:
`stop()` doesn't need to be executed again if the HAP server is already stopped, because register service failed.

Reply:
Ah I see, yes